rust/blockstore/src/arrow/block/types.rs

```diff
@@ -1,6 +1,6 @@
 use super::delta::BlockDelta;
 use crate::arrow::types::{ArrowReadableKey, ArrowReadableValue};
-use arrow::array::ArrayData;
+use arrow::array::{ArrayAccessor, ArrayData};
```

Comment:
unused import

comment is straightforward

i am sitting next to you at 660 alabama